MovieDB/MovieDB/Scenes/Dashboard/DashboardCoordinator.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+
+protocol DashboardCoordinatorType {
```

Comment:
You should conform the Coordinator protocol here. It should be better than the DashboardCoordinator conform directly Coordinator